src/views/Root/index.js

```diff
@@ -15,17 +15,17 @@ const Root = () => (
       <Route exact path="/profile">
         <Profile />
       </Route>
-      <Route exact path={['/', '/home']}>
+      <Route path={['/', '/home']}>
         <Header>
           <Home />
         </Header>
       </Route>
-      <Route exact path="/dashboard">
+      <Route path="/dashboard">
         <Header>
           <Dashboard />
         </Header>
       </Route>
-      <Route exact path="/details/:placeId">
```

Comment:
Why not `exact` attribute?

Reply:
If we want to get something like on image belowe we have to let router reach e.g `/home` and `/home/signin` at the same time
![image](https://user-images.githubusercontent.com/61630877/108124372-66b63a80-70a7-11eb-84c0-143845916172.png)

Reply:
I think that sign in view will be display on foreground

Reply:
Ok but we still have `/search` route that is displaying with `/home` and `/dashboard` route.